projects/laji/src/app/shared/model/Form.ts

```diff
@@ -37,10 +37,7 @@ export namespace Form {
     options: ListOptions;
   }
 
-  export interface IEnum {
-    enum: string[];
-    enumNames: string[];
-  }
+  export type IEnum = {const: string; title: string}[];
```

Comment:
Should this interface be `oneOf: {const: string; title: string}[];` (like in the file `projects/laji-api-client/src/lib/models/UISchemaContext.ts` above)?

Reply:
> Because there are changes to label designer, we must remember to update the label designer npm package before the update

Right. The label designer schema service is used only on laji.fi, not on Kotka right? Does laji.fi use the label-designer from the npm package, or from directly from the source? If it uses the label designer from npm, we have to make a release for it and include a dependency update to this PR... It's in package.json deps at least so I assume we have to do it that way. So this PR is recursively depending on it's changes. Pretty crazy stuff!

Reply:
That's correct but [this pull request](https://github.com/luomus/laji/pull/337) needs to be merged first because I made release 4.0.6 for label designer there

Reply:
Ah, so I'll merge that to dev now and then merge dev here.